src/stache/includes/cli/browser.html

```diff
@@ -1 +1 @@
-<sky-code>--browser</sky-code> or <sky-code>-b</sky-code> — Specifies the browser to use. This option is useful for cross-browser manual testing. The available values are platform-dependent, and they correspond to the names of the browser executables on each platform. For example, the value for Chrome is <sky-code>google chrome</sky-code> on macOS, <sky-code>google-chrome</sky-code> on Linux, and <sky-code>chrome</sky-code> on Windows. Microsoft Edge is an exception that is not platform-dependent, and its value is <sky-code>edge</sky-code> on all platforms. You can also pass in the browser's full path. SKY UX passes the value to the <sky-code>open</sky-code> library as the <sky-code>app</sky-code> property, so for more information, see <a href="https://github.com/sindresorhus/open#app">the <sky-code>open</sky-code> library's README file</a>.
\ No newline at end of file
+<sky-code>--browser</sky-code> or <sky-code>-b</sky-code> — Specifies the browser to use. This option is useful for cross-browser manual testing. The available values are platform-dependent, and they correspond to the names of the browser executables on each platform. For example, the value for Chrome is <sky-code>google chrome</sky-code> on macOS, <sky-code>google-chrome</sky-code> on Linux, and <sky-code>chrome</sky-code> on Windows. Microsoft Edge is an exception that is not platform-dependent, and its value is <sky-code>edge</sky-code> on all platforms. You can also pass in the browser's full path. SKY UX passes the value to the <sky-code>open</sky-code> library as the <sky-code>app</sky-code> property, so for more information, see <a href="https://github.com/sindresorhus/open#app">the <sky-code>open</sky-code> library's README file</a>. You can specify multiple <sky-code>--browser</sky-code> arguments. The first argument specifies the browser to use, while subsequent arguments are passed to the browser. This is useful for serving the application in incognito/private mode. For example, to run the application in Chrome on Linux and use inincognito mode, enter <sky-code>skyux server --browser='google-chrome' --browser='--incognito'</sky-code>.
```

Comment:
Last sentence says `skyux server` instead of `skyux serve`.

Reply:
Fixed the typo. Thanks!